components/sections/Buy.js

```diff
@@ -0,0 +1,130 @@
+import Link from "next/link";
+
+
+export default function Wallets() {
```

Comment:
Not required, but, might be nice to call the function Buy() instead and use a variable called 'exchanges' or something instead of 'wallets'.